src/Chapter1.hs

```diff
@@ -568,7 +575,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c 
+      | (c == 'a' || c == 'e' || c == 'i' || c == 'o' ||  c == 'u') = True
+      | otherwise = False
```

Comment:
Nice one! 👍🏼 
As `==` and `||` return `Bool` itself, you can just return the result of this function:

```haskell
isVowel c = c == 'a' || c == 'e' || c == 'i' || c == 'o' ||  c == 'u'
```

Reply:
Thanks for this elegant and simple alternative